client/X11/xf_client.c

```diff
@@ -1077,7 +1077,8 @@ static const button_map xf_button_flags[NUM_BUTTONS_MAPPED] = {
 	{ Button3, PTR_FLAGS_BUTTON2 },
 	{ Button4, PTR_FLAGS_WHEEL | 0x78 },
 	{ Button5, PTR_FLAGS_WHEEL | PTR_FLAGS_WHEEL_NEGATIVE | 0x78 },
```

Comment:
Seems like that you need to force push to this branch again :D You reverted now the complete change for Button5. `0x78` needs to be replaced here with `(0x100 - 0x78)`.

Reply:
looks like I am blind today :/